src/main/java/dev/snowdrop/release/App.java

```diff
@@ -200,7 +200,7 @@ public void startRelease(
         }
 
         // link CVEs
-        for (var cve : cveService.listCVEs(Optional.of(release.getVersion()), false)) {
+        for (var cve : cveService.listCVEs(Optional.of(release.getVersion().replace(".RELEASE", "")), false)) {
```

Comment:
We should unify how release strings are passed around and validate them.

Reply:
I have removed this because the solution is not as simple as this. It requires a broader look.